### ISSUE TITLE
`no-raw-locators` no longer flags interpolated template-literal locators

### ISSUE THREAD

**Issue by hawkrives** [playwright-community/eslint-plugin-playwright#479]
Since `fix: Support nested locators everywhere` (0e481868), `no-raw-locators` has stopped flagging `.locator()` calls if we call it with a template literal _with interpolation_. Only strings and template strings without interpolation are still flagged. 

```ts
// eslint.config.mjs
import playwright from 'eslint-plugin-playwright';
export default [{
  files: ['test.mjs'],
  plugins: { playwright },
  rules: { 'playwright/no-raw-locators': 'warn' },
}];
```

```ts
// test.mjs
const id = '123';
page.locator(`#row-${id}`);
```

- `eslint-plugin-playwright@2.2.2`: reports `Usage of raw locator detected...` ✅
- `eslint-plugin-playwright@2.10.5`: no report ❌
